### DISCUSSION TITLE
avg_slopes() and AER package Tobit

### DISCUSSION THREAD

**Discussion by arantzagorostiaga** [vincentarelbundock/marginaleffects#1141] [Q&A] [answered]
Hi everyone

I have an issue when calculating avg_slopes() with the ouptut of tobit() function from the AER package. 

When I run avg_slopes(), the average marginal effects are exactly equal to the tobit coefficients. It seems that only the linear part of the model is considered, rather than the full Tobit model.

Here’s an example:

    library(AER)
    library(marginaleffects)
    data <- read.csv("https://vincentarelbundock.github.io/Rdatasets/csv/AER/Affairs.csv")

    model <- tobit(
      affairs ~ age + yearsmarried + religiousness + occupation + rating,
      data = data,
      x = TRUE)

    summary(model)

    avg_slopes(model)

I expected the output to be:

    mean(pnorm(model$x %*% model$coef / model$scale)) * model$coef[-1]

What am I missing here?

**Comment by vincentarelbundock** [comment]
Can't try right now. But did you check different values of the type argument?

**Reply by vincentarelbundock** [reply, marked as answer]
May be relevant: https://stats.stackexchange.com/a/149529/4874

**Reply by arantzagorostiaga** [reply]
Many thanks for your prompt reply!  I understand now, it's the effect on the conditional mean of the latent variable which is what predict() returns.

Thanks again for your help and for the great marginaleffects package.
